engine/admin/logging/logentries.md

```diff
@@ -0,0 +1,51 @@
+---
+aliases: ["/engine/reference/logging/logentries/"]
```

Comment:
It looks like it's probably fine, it just turns it into a collection of one. I think generally we write a single one inline, but multiple ones as a list:

```yaml
aliases: "/engine/reference/logging/logentries/"
aliases:
  - "/engine/reference/logging/logentries/"
  - "/engine/reference/logging/logentries/"
```

I don't think it will break anything either way.